man/systemd.netdev.xml

```diff
@@ -343,8 +343,16 @@
             </para>
           </listitem>
         </varlistentry>
+          <varlistentry>
+          <term><varname>EnableSTP=</varname></term>
```

Comment:
I think this should just be called "STP=", we don't prefix any other boolean options with "Enable", and hence we shouldn't do this for this one either.